web/src/engine/websites/legacy/ComicZerosum.ts

```diff
@@ -1,56 +1,94 @@
-// Auto-Generated export from HakuNeko Legacy
-// See: https://gist.github.com/ronny1982/0c8d5d4f0bd9c1f1b21dbf9a2ffbfec9
-
-//import { Tags } from '../../Tags';
+import { Tags } from '../../Tags';
 import icon from './ComicZerosum.webp';
-import { DecoratableMangaScraper } from '../../providers/MangaPlugin';
+import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../../providers/MangaPlugin';
+import * as Common from '../decorators/Common';
+import { FetchProto, FetchRequest } from '../../FetchProvider';
 
-export default class extends DecoratableMangaScraper {
+type APISingleTitle = {
+    title: APITitle
+}
+
+type APITitles = {
+    titles: APITitle[]
+}
+
+type APITitle = {
+    id: number,
+    tag: string,
+    name: string
+}
+
+type TitleView = {
+    title: APITitle,
+    chapters: APIChapter[]
+}
+
+type APIChapter = {
+    id: number,
+    name: string,
+    imageUrl: string
+}
+
+type APIPages = {
+    pages: {
+        imageUrl: string
+    }[]
+}
 
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+    private api_url = 'https://api.zerosumonline.com/api/v1/';
+    private protoTypes = '/src/engine/websites/legacy/ComicZerosum.proto';
```

Comment:
I don't think this path will be available when bundled and deployed.
The proto types probably needs to be bundled as well...

Reply:
I knew that and i asked myself if  we could have done something about the path problem. Oh well, prototypes are strings included in the ts file now. Changed FetchProto accordingly.